staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -52,6 +52,8 @@ type DeltaFIFOOptions struct {
 	// When false, `Sync` events will be sent instead.
 	EmitDeltaTypeReplaced bool
 
+	EmitDeltaTypeBookmark bool
```

Comment:
add godoc describing what this does and what the "object" in the emitted deltas is (string resourceVersion)

Reply:
Done